openaddresses/be/vlg/bosa-be-fr.json

```diff
@@ -0,0 +1,39 @@
+{
+    "data": "FIXME insert correct url here",
```

Comment:
Also include the URL here (https://opendata.bosa.be/download/best/openaddress-bebru.zip)

Reply:
These files are for the flemish region, so i made the links:
* BE-VLG -->  https://opendata.bosa.be/download/best/openaddress-bevlg.zip
* BE-WAL --> https://opendata.bosa.be/download/best/openaddress-bewal.zip